tests/test_pypi.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from scikit_package.utils.pypi import (  # Replace with your actual module
+    check_pypi_package_exists,
+)
+
+
+def test_package_exists():
```

Comment:
we generally don't want to run actual requests in our tests but mock them.  Anything else is too brittle (tests pass or fail depending on internet connection for example)  We assume that the requests package and the API end-points are tested by their own developers.  So here we are just testing our local behavior....when it works, print the right thing, when it fails, raise the right expeption, print the right message.

Please use mock and not monkeypatch for uniformity across projects.

Reply:
yes, done using `pytest-mock`